templates/client-config-secret.conf.j2

```diff
@@ -9,6 +9,9 @@ openshift_url = {{ cluster.openshift_url }}
 token_file = /var/run/secrets/atomic-reactor/{{ cluster.name | replace('_', '-') }}-orchestrator/token
 use_auth = {{ cluster.use_auth | default('true') }}
 verify_ssl = {{ cluster.verify_ssl | default('true') }}
+{% if cluster.auto_build_node_selector %}
```

Comment:
Won't this cause an exception if `cluster` does not have `auto_build_node_selector` attribute? Might need to use `default('')` filter here.

Reply:
ok changed